packages/datagateway-dataview/public/res/default.json

```diff
@@ -211,7 +211,10 @@
   "buttons": {
     "add_to_cart": "Add to selection",
     "remove_from_cart": "Remove from selection",
-    "download": "Download"
+    "download": "Download",
+    "tooltip": {
```

Comment:
Not sure on the structure of the language file here - perhaps getting rid of the `tooltip` level and just having `unable_to_download_tooltip` is clearer?

Reply:
Very fair, I flip-flopped on this one too